### ISSUE TITLE
Question about training on High-level Model

### ISSUE THREAD

**Issue by Zhimin-C** [suhas-srinath/GRepQ#2]
Thanks for the great work! I trained the high-level model by changing the run type in the configuration to hl_model_train. However, when I directly evaluate the high-level model's zero-shot performance on the CLIVE dataset, it only achieves scores of around 66 and 70. In contrast, the provided pre-trained model achieves scores of 74 and 76. Could you please let me know if there are any additional steps or specific configurations required to train the high-level model? Thanks!

![Screenshot 2024-06-06 at 4 13 09 PM](https://github.com/suhas-srinath/GRepQ/assets/35618385/dae1629b-ea92-4864-8a4e-8871fc7a2df2)

**Comment by suhas-srinath** [comment]
@Zhimin-C Sorry for the delayed reply. The hyperparameters you can try are - learning rate 1e-5, Adam optimizer, group size 8, batch size 128 (which is a separation of 96 images within the batch) and 15 epochs.
